arroyo-worker/src/connectors/sse.rs

```diff
@@ -190,11 +191,13 @@ where
                                             for v in iter {
                                                 match v {
                                                     Ok(value) => {
-                                                        ctx.collector.collect(Record {
-                                                            timestamp: SystemTime::now(),
-                                                            key: None,
-                                                            value,
-                                                        }).await;
+                                                        if let Some(value) = value {
```

Comment:
The HTTP sources (SSE and websocket) have their own implementation of this that we probably want to replace